src/Microsoft.Azure.SignalR.Common/Endpoints/AccessKey.cs

```diff
@@ -1,10 +1,14 @@
-﻿namespace Microsoft.Azure.SignalR
+﻿using System.Threading.Tasks;
+
+namespace Microsoft.Azure.SignalR
 {
     internal class AccessKey
     {
         public string Value { get; set; }
         public string Id { get; set; }

```

Comment:
>set; [](start = 32, length = 4)

Make `Value` and `Id` get only? Whenever Value changes, shall we view it as a new AccessKey?